packages/utils/src/string.ts

```diff
@@ -112,42 +120,49 @@ export function isMatchingPattern(value: string, pattern: RegExp | string): bool
  * @returns A base64-encoded version of the string
  */
 export function unicodeToBase64(plaintext: string): string {
-  const global = getGlobalObject();
-
-  // Cast to a string just in case we're given something else
-  const stringifiedInput = String(plaintext);
-  const errMsg = `Unable to convert to base64: ${
-    stringifiedInput.length > 256 ? `${stringifiedInput.slice(0, 256)}...` : stringifiedInput
-  }`;
+  const globalObject = getGlobalObject() as GlobalWithBase64Helpers;
```

Comment:
```suggestion
  const globalObject = getGlobalObject<GlobalWithBase64Helpers>();
```

This should work as well

Reply:
Oh, nice. I'd forgotten that the function itself does a cast to `T`.